tests/specs/benchmarks/verification.k

```diff
@@ -385,26 +341,43 @@ module VERIFICATION-JAVA [symbolic, kast]
     claim <k> ECREC => #end EVMC_SUCCESS ... </k>
          <callData> DATA </callData>
          <output> _ => #ecrec(#symEcrec(DATA)) </output>
-      requires notBool #isConcrete(DATA) andBool #sizeByteArray(DATA) ==Int 128 andBool notBool #ecrecEmpty(DATA)
+      requires notBool #isConcrete(DATA)
+       andBool #sizeByteArray(DATA) ==Int 128
+       andBool notBool #ecrecEmpty(DATA)
       [trusted]
 
     claim <k> ECREC => #end EVMC_SUCCESS ... </k>
          <callData> DATA </callData>
          <output> _ => #ecrec(.Account) </output>
-      requires notBool #isConcrete(DATA) andBool #sizeByteArray(DATA) ==Int 128 andBool #ecrecEmpty(DATA)
+      requires notBool #isConcrete(DATA)
+       andBool #sizeByteArray(DATA) ==Int 128
+       andBool #ecrecEmpty(DATA)
       [trusted]
 
   // ########################
   // Arithmetic
   // ########################
 
     rule ((A +Int I1) +Int B) +Int I2 => (A +Int B) +Int (I1 +Int I2)
-      requires notBool #isConcrete(A) andBool notBool #isConcrete(B) andBool #isConcrete(I1) andBool #isConcrete(I2) [simplification]
+      requires notBool #isConcrete(A)
+       andBool notBool #isConcrete(B)
+       andBool #isConcrete(I1)
+       andBool #isConcrete(I2)
+      [simplification]
 
     rule (A +Int I1) +Int (B +Int I2) => (A +Int B) +Int (I1 +Int I2)
-      requires notBool #isConcrete(A) andBool notBool #isConcrete(B) andBool #isConcrete(I1) andBool #isConcrete(I2) [simplification]
+      requires notBool #isConcrete(A)
+       andBool notBool #isConcrete(B)
+       andBool #isConcrete(I1)
+       andBool #isConcrete(I2)
+      [simplification]
 
-    rule chop(X +Int Y) => X +Int Y requires 0 <=Int X andBool X <=Int pow16 andBool 0 <=Int Y andBool Y <Int pow16 [simplification]
+    rule chop(X +Int Y) => X +Int Y
+      requires 0 <=Int X
+       andBool X <=Int pow16
```

Comment:
For cases like this, it may look better to write like either:

```
0 <=Int X andBool X <=Int pow16
```

Or you can use the syntactic sugar we have which is:

```
#range(0 <= X <= pow16)
```

But either way I'd like to see the range that X is allowed to span on a single line.